### DISCUSSION TITLE
Chrome file size

### DISCUSSION THREAD

**Discussion by Valk013** [Eloston/ungoogled-chromium#2056] [Q&A]
Hello. I've noticed that the "Chrome" file size is increased significantly in the last version 104.0.5112.102 (compared to 103.0.5060.134).
But it didn't increase so much in the last (same) version of Google Chrome.
Who can explain that?

**Comment by networkException** [comment]
Which file exactly?

**Reply by Valk013** [reply]
File name is Chrome.
It's about Linux binaries downloaded from https://ungoogled-software.github.io/ungoogled-chromium-binaries/releases/linux_portable/64bit/

**Reply by networkException** [reply]
I don't know why that would happen, maybe @clickot has an explanation? Any changes to your build system?

**Comment by clickot** [comment]
I made no changes on my build system. On my system, in version 103.0.5060.134-1 the binary has a size of 296M, in 104.0.5112.81-1 317M and 319M in the latest version, so it seems the binary size made the significant increase from 103.xx to 104.xx, but i don't know why. In all cases the build procedure and the used docker image (bullseye with current clang-15) was the same (except i update all packages before build)
